src/mail/view/MailViewer.ts

```diff
@@ -727,6 +729,25 @@ export class MailViewer implements Component<MailViewerAttrs> {
 		}
 		return false
 	}
+
+	private async handleAttachmentImport(file: TutanotaFile) {
```

Comment:
from a first glance all of this seems to be mostly unrelated to `MailViewer`. I wonder if we could do  something like
`Mailviewer -> MailViewerViewModel.importContactFromFile() -> ContactImporter.importContactFromFile()` and do all of this there?

Reply:
Part of it can be moved to another file, but downloading & parsing the file needs to be done here as we enforce the sessionkey update to the file based on the email.

Reply:
I don't think parsing needs to be done here, only downloading/enforcing session key stuff and then we can pass it on to importer